src/localization/ru/learn.json

```diff
@@ -0,0 +1,163 @@
+{
+  "examples.starter.title": "Практика",
+  "examples.starter.description": "Теперь пора попрактиковаться, чтобы закрепить то, что вы узнали.",
+
+  "examples.finish.title": "Практика завершена!",
+  "examples.finish.description": " ",
+
+  "steps.starter.title": "Вступление",
+  "steps.starter.description": "RegEx - это сокращение от `Regular Expression`. Это помогает сопоставлять, находить или управлять текстом. Начните с ввода `OK` в поле RegEx, чтобы перейти к первому шагу и получить доступ к более подробному описанию..",
+
+  "steps.whatIsRegex.title": "Что такое регулярные выражения `RegEx`?",
+  "steps.whatIsRegex.description": "Регулярные выражения - это строка символов, которые выражает шаблон поиска, часто сокращается как `RegEx` или `RegExp`. RegExp используется для поиска или замены слов в текстах, кроме того, мы можем проверить, соответствует ли текст установленным нами правилам или нет.\\n\\n Например, у вас есть список имен файлов и вы хотите найти только файлы с расширением `pdf`. После ввода выражения `^\\w+\\.pdf$` это будет работать. Значение определений в этом выражении станет яснее по мере выполнения шагов.",
+
+  "steps.basicMatchers.title": "Базовые сходства",
+  "steps.basicMatchers.description": "Символ или слово, которое мы хотим найти, написано напрямую - это похоже на обычный процесс поиска. Например, чтобы найти слово `curious` в тексте, введите то же самое.",
+
+  "steps.dotCharacter.title": "Точка `.`: Любые символы",
+  "steps.dotCharacter.description": "Точка `.` позволяет выбрать любой символ, включая специальные символы и пробелы.",
+
+  "steps.characterSet.title": "Наборы символов `[abc]`",
+  "steps.characterSet.description": "Если одним из символов в слове могут быть разные символы, мы записываем его в квадратные скобки `[]` со всеми альтернативными символами. Например, чтобы написать выражение, которое может найти все слова в тексте, введите символы `a`,` e`, `i`,` o`, `u` в квадратных скобках `[]`.",
```

Comment:
Some spaces are in the wrong position.
```js
`a`,` e`, `i`,` o`, `u`
```
It should be like this
```js
`a`, `e`, `i`, `o`, `u`
```

Reply:
Solved.